tizen-api/doc/nnstreamer_doc.h

```diff
@@ -52,21 +52,21 @@
  * @section CAPI_ML_NNSTREAMER_PIPELINE_OVERVIEW Overview
  * The NNStreamer API provides interfaces to create and execute stream pipelines with neural networks and sensors.
```

Comment:
is it alright to have API here?

Reply:
`check-header.py` of tizen-native-api-review-script does not catch this `API` word. but you are right. I will fix it ASAP.